login/Models/RoleModel.js

```diff
@@ -1,6 +1,6 @@
-const sq =  require("../DataConnection.js");
-const sequelize = sq.sequelize;
-const Sequelize = sq.Sequelize;
+const dataconnection =  require("../DataConnection.js");
+const sequelize = dataconnection.sequelize;
+const Sequelize = sequelize.Sequelize;
```

Comment:
These two sequelize variables with different cases are confusing to me.

Reply:
sorry,i fixed it but had trouble with GIT